Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -265,6 +265,122 @@ using TrackExtra = TracksExtra::iterator;
 using FullTracks = soa::Join<Tracks, TracksCov, TracksExtra>;
 using FullTrack = FullTracks::iterator;
 
+namespace trackfwd
+{
+// TRACKPARFWD TABLE definition
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+DECLARE_SOA_COLUMN(TrackType, trackType, uint8_t); // TODO change to TrackTypeEnum when enums are supported
```

Comment:
Please create an enum ForwardTrackTypes for the types in DataTypes.h
then change the name of the enum in the comment

Reply:
Done. @aphecetche , @pillot notice alias MCHStandalone for Run2-like MUON Tracks in DataTypes.h.

Reply:
Thanks. I understand we will have also MCHStandalone for Run 3 ? Otherwise (if it is just legacy) we should add Run2 in the name (as in the other enum) and put it at the end.

Reply:
@jgrosseo yes. It is not just legacy in the sense that also in run3 some analysis will use these tracks and not the ones with MFT, in order for instance to profit from the full MUON acceptance (MFT acceptance is smaller).